main.js

```diff
@@ -38,31 +38,31 @@ const removeClass = (el, oldClass) => {
 
     if (matchedObjects.length == 0) return;
 
-    matchedObjects.map((obj) => {
-      matchedEmojiTitles.push(obj.keywords[0]);
-    });
-
-    for (const emoji of matchedEmojiTitles) {
-      const emojiItem = document.querySelector(`[title="${emoji}"]`);
-
-      if (emojiItem) {
-        addClass(emojiItem.parentElement,'visible');
-      }
+    for (const matched of matchedObjects) {
+      setMatchedEmojiVisible(matched);
```

Comment:
Any reason why we can't just resolve the merge conflicts with master?

When we use this refactor as the last exercise, we'll have to show why the
function was necessarily. (Because it's used for destructuring). That means
your branch shouldn't have that function for the earlier exercise.

![](https://link.nylas.com/open/6hy0p9pbfmrqd5xtd6chwq447/local-cbd07d4b-
0ac7?r=cmVwbHkrMDA1NjQyODk4Mzg3MDUzNDBkNmRhYTlhNzE2YzE0OTZjYzIzNThlODk0NTA5MzViOTJjZjAwMDAwMDAxMTNiMGE5Nzc5MmExNzBjZTA0NTFlZDdhQHJlcGx5LmdpdGh1Yi5jb20=)